sync/index.markdown

```diff
@@ -23,28 +23,28 @@ Below are the various GitHub repositories that are part of the Data-Sync feature
 
 ### Data-Sync Server
 
-Our Netty-based [Java Server](https://github.com/aerogear/aerogear-sync-server) exposes WebSocket and XMPP/GCM endpoint for our different clients.
+* [Java Sync Server Engine](https://github.com/aerogear/aerogear-sync-server/tree/master/server/server-core#aerogear-server-differential-synchronization-server-core) the Server Sync Engine.
+* [Netty Sync Server](https://github.com/aerogear/aerogear-sync-server/tree/master/server/server-netty#netty-differential-synchronization-server-implementation) which exposes WebSocket and XMPP/GCM endpoint for our different clients.
```

Comment:
> but it might not hurt pointing this out.

yeah, I agree

On Mon, Feb 23, 2015 at 12:22 PM, Daniel Bevenius <notifications@github.com>
wrote:

> In sync/index.markdown
> <https://github.com/aerogear/aerogear.org/pull/498#discussion_r25156981>:
>
> > @@ -23,28 +23,28 @@ Below are the various GitHub repositories that are part of the Data-Sync feature
> >
> >  ### Data-Sync Server
> >
> > -Our Netty-based [Java Server](https://github.com/aerogear/aerogear-sync-server) exposes WebSocket and XMPP/GCM endpoint for our different clients.
> > +* [Java Sync Server Engine](https://github.com/aerogear/aerogear-sync-server/tree/master/server/server-core#aerogear-server-differential-synchronization-server-core) the Server Sync Engine.
> > +* [Netty Sync Server](https://github.com/aerogear/aerogear-sync-server/tree/master/server/server-netty#netty-differential-synchronization-server-implementation) which exposes WebSocket and XMPP/GCM endpoint for our different clients.
>
> I was not sure if we should or not. If you follow the link I think this is
> clear but it might not hurt pointing this out.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/498/files#r25156981>.
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf